src/index.html

```diff
@@ -4,14 +4,176 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>HTML</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+
+    <form class="form" onsubmit="onSubmit()">
+
+      <script type="text/javascript" src="./main.js"></script>
+
+      <fieldset class="first-block" aria-autocomplete="none">
+        <legend class="opo">Personal information</legend>
+
+        <div class="x">
+          <label for="box-s">
+            Surname:
+          </label>
+          <input type="text" id="box-s">
+        </div>
+
+        <div class="x">
+          <label for="box-n">
+            Name:
+          </label>
+          <input type="text" id="box-n">
+        </div>
+
+        <div class="x">
+          <label for="box-h">
+            How old are You?
+          </label>
+          <input
+            type="number"
+            value="12"
+            id="box-h"
+            min="12"
+            max="100"
+          >
+        </div>
+
+        <div class="x">
+          <label for="box-f">
+            Full date of birth:
+          </label>
+          <input
+            type="date"
+            value="дд.мм.рррр"
+            id="box-f"
+          >
+        </div>
+
+        <div class="y">
+          <label for="box-i">
+            I accept the term of the agreement
+          </label>
+          <input type="checkbox" id="box-i">
+        </div>
+      </fieldset>
+
+
+      <fieldset class="second-block" aria-autocomplete="none">
+        <legend>Registration</legend>
+        <div class="x">
+          <label for="box-e">
+            E-mail:
+          </label>
+          <input type="email"
+    id="box-e"
+    placeholder="email@example.com"
+    multiple>
```

Comment:
Отформатируй код правильно
```suggestion
          <input 
            type="email"
            id="box-e"
            placeholder="email@example.com"
            multiple
          >
```